src/usr/share/batwatch/batwatch.py

```diff
@@ -15,162 +13,179 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-"""
-Handles battery change status monitoring and e-mail notifications for Batwatch.
+"""Handles battery change status monitoring and e-mail notifications for the battery
+monitor.
 """
 
-__author__ = 'Joel Luellwitz, Emily Klapp, and Brittney Scaccia'
+__all__ = ['BatWatch']
+__author__ = 'Joel Luellwitz, Emily Frost, and Brittney Scaccia'
 __version__ = '0.8'
 
 import logging
+import random
 import time
-import gpgmailmessage
 from pydbus import SystemBus
+import gpgmailmessage
 
 UPOWER_BUS_NAME = 'org.freedesktop.UPower'
-FULLY_CHARGED, CHARGING, DISCHARGING, NO_BATTERY = range(4)
-CHARGE_STATUS_DICT = {
-    0: 'Fully Charged',
-    1: 'Charging',
-    2: 'Discharging',
-    3: 'No Battery'
-}
+UPOWER_DEVICE_STATE_CHARGING = 1
+UPOWER_DEVICE_STATE_FULLY_CHARGED = 4
 
+# These constants are integers instead of strings because they need to be compared to
+#   determine "favorable" changes.
+NO_BATTERY, DISCHARGING, CHARGING, FULLY_CHARGED = range(4)
+
+# This list is built with insert instead of statically so that we only need to maintain
+#   the order in one place.
```

Comment:
Replace with "This list is built with insert instead of statically to avoid maintaining two lists with consistent ordering."